functions/index.js

```diff
@@ -0,0 +1,72 @@
+
+const functions = require('firebase-functions');
+var admin = require("firebase-admin");
+
+
+admin.initializeApp({
+    credential: admin.credential.applicationDefault(),
+    databaseURL: "https://audio-seva-team-test.firebaseio.com",
+    storageBucket: 'audio-seva-team-test.appspot.com'
+});
+
+
+exports.on_mp3_upload = functions.storage.object().onFinalize( object => {
```

Comment:
Name functions according to the action, not the trigger. I think it might be better. As later we can add another action to the same trigger as a different function.

Reply:
checked